Source/JavaScriptCore/runtime/JSRunLoopTimer.h

```diff
@@ -65,7 +66,8 @@ class JSRunLoopTimer : public ThreadSafeRefCounted<JSRunLoopTimer> {
         Lock m_lock;
 
         class PerVMData {
-            WTF_MAKE_FAST_ALLOCATED;
+            WTF_MAKE_FAST_ALLOCATED; // mlam FIXME
+//            WTF_MAKE_TZONE_ALLOCATED(PerVMData);
```

Comment:
Please remove the FIXME and commented out line of code.

You should just grep for "mlam" and remove all such comments.

Reply:
Fixed.